src/app/containers/CpsAssetPageMain/index.jsx

```diff
@@ -1,14 +1,31 @@
 import React from 'react';
 import { string, shape, object, arrayOf } from 'prop-types';
+import path from 'ramda/src/path';
+
 import { Grid, GridItemConstrainedMedium } from '#lib/styledGrid';
+import MetadataContainer from '../Metadata';
+import LinkedData from '../LinkedData';
+
+const CpsAssetPageMain = ({ pageData }) => {
+  const title = path(['promo', 'headlines', 'headline'], pageData);
+  const summary = path(['promo', 'summary'], pageData);
+  const metadata = path(['metadata'], pageData);
 
-const CpsAssetPageMain = () => {
   return (
-    <Grid as="main" role="main">
-      <GridItemConstrainedMedium>
-        <h1> Placeholder content for MAP page skeleton</h1>
-      </GridItemConstrainedMedium>
-    </Grid>
+    <>
+      <MetadataContainer
+        title={title}
+        lang={metadata.language}
+        description={summary}
+        openGraphType="website"
+      />
+      <LinkedData type="Article" seoTitle={title} />
```

Comment:
is this container only going to be used for Articles? Could the hardcoding of `article` become a problem here

Reply:
I believe Article is the correct type for MAP and STY pages - not sure what other page types are going to be rendered by this

I do have some doubt about "Article" being correct for MAPs - it seems correct technically as MAPs are just articles with a video at the top, but there seems to be a sentiment that editorial don't actually care about the article content below the media - so Article might not be correct

cc @aeroplanejane

Reply:
Ok thanks, I was thinking about the new feature indexes (FIX assets) however at the moment they are using the FrontPage container

Reply:
Just had a meeting with Tom Chandler on this, and there's a lot of work needing analysed and implemented for this - different schemas per page, and per media type (audio MAPs have different schemas from video MAPs)

I'd recommend going with this for now (it's better than what we currently have - nothing) and creating an issue to track the v2 improvements